test/language_server/run_langserv.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
I really like these tests! Sadly Gleam aims to be totally cross platform so bash isn't an ideal scripting language. Could we convert these to run using NodeJS or similar? That was windows users can run the tests and we can run these on our Windows CI build.

Reply:
These tests being left in were a mistake!  The intention was to replace them wholly with tests in rust using [tower-test](https://docs.rs/tower-test/0.3.0/tower_test/index.html)